src/libs/getTL.js

```diff
@@ -0,0 +1,15 @@
+/* Data get information */
+export async function getDataTLFromId(text, city = false) {
+  const substring = city ? `explore/locations/${text}` : text;
+  const url = `https://www.instagram.com/${substring}/?__a=1`;
+  try {
+    const response = await fetch(url);
+    if (response && response.url && response.url.includes("accounts/login")) {
```

Comment:
When IG responds with no JSON, this won't break anymore thanks to this line. At least, for the `accounts/login` case for now